packages/webcomponents/src/components/checkout/new-payment-method.tsx

```diff
@@ -86,7 +86,8 @@ export class NewPaymentMethod {
     }
   }
 
-  onPaymentMethodOptionClick = () => {
+  onPaymentMethodOptionClick = (e) => {
+    e.preventDefault();
```

Comment:
just curious about the reason for this change?

Reply:
The click event was bubbling up and introducing some weirdness